### DISCUSSION TITLE
How does observe work?

### DISCUSSION THREAD

**Discussion by Inferdy** [mxgmn/MarkovJunior#34] [Q&A] [answered]
I don't understand it completely.

**Comment by GroophyLifefor** [comment]
We are too waiting one of simple explain.

https://github.com/mxgmn/MarkovJunior/discussions/22

**Comment by mxgmn** [comment, marked as answer]
I just wrote about `observe` elements in [syntax.md](https://github.com/mxgmn/MarkovJunior/blob/main/syntax.md).
